src/rust/ide/src/view/layout.rs

```diff
@@ -93,9 +93,10 @@ impl ViewLayout {
     , text_controller          : controller::Text
     , graph_controller         : controller::ExecutedGraph
     , visualization_controller : controller::Visualization
-    , project                  : Rc<model::Project>
+    , project                  : impl Into<Rc<dyn model::project::API>> // TODO[ao]!!!!!!!!!!!!
```

Comment:
Uhm, well, left TODO

Reply:
use `model::Project`
and link to issue https://github.com/rust-lang/rust/issues/63033